README.md

```diff
@@ -0,0 +1,67 @@
+- `AC_SL_ENV`: An environment variable key value pair that may be referenced in the tests executed by this command. Expanded environment variables are supported. Check [saucectl](https://docs.saucelabs.com/dev/cli/saucectl/run/#--env) documentation for the option detail.
+- `AC_SL_FAIL_FAST`: Stop suite execution after the first failure. Check [saucectl](https://docs.saucelabs.com/dev/cli/saucectl/run/#--fail-fast) documentation for the option detail.
+- `AC_SL_RETRIES`: Number of times to rerun a failed test suite. Check [saucectl](https://docs.saucelabs.com/dev/cli/saucectl/run/#--retries) documentation for the option detail.
+- `AC_SL_ROOT_DIR`: Specifies the project directory. Check [saucectl](https://docs.saucelabs.com/dev/cli/saucectl/run/#--root-dir) documentation for the option detail.
+- `AC_SL_SAUCEIGNORE`: Specifies the path to the `.sauceignore` file. Check [saucectl](https://docs.saucelabs.com/dev/cli/saucectl/run/#--sauceignore) documentation for the option detail.
+- `AC_SL_SHOW_CONSOLE_LOG`: Include the `console.log` contents in the output for all tests. Check [saucectl](https://docs.saucelabs.com/dev/cli/saucectl/run/#--show-console-log) documentation for the option detail.
+- `AC_SL_TAGS`: Keywords that may help you distinguish the test in Sauce Labs, and also help you apply filters to easily isolate tests based on metrics that are meaningful to you. Check [saucectl](https://docs.saucelabs.com/dev/cli/saucectl/run/#--tags) documentation for the option detail.
+- `AC_SL_TIMEOUT`: Global timeout that limits how long saucectl can run in total. Check [saucectl](https://docs.saucelabs.com/dev/cli/saucectl/run/#--timeout) documentation for the option detail.
+- `AC_SL_TUNNEL_NAME`: Use a running Sauce Connect tunnel to test. Check [saucectl](https://docs.saucelabs.com/dev/cli/saucectl/run/#--tunnel-name) documentation for the option detail.
+- `AC_SL_TUNNEL_OWNER`: The tunnel owner, if it is not the testing account. Check [saucectl](https://docs.saucelabs.com/dev/cli/saucectl/run/#--tunnel-owner) documentation for the option detail.
+- `AC_SL_TUNNEL_TIMEOUT`: How long to wait for the specified tunnel to be ready. Check [saucectl](https://docs.saucelabs.com/dev/cli/saucectl/run/#--tunnel-timeout) documentation for the option detail.
+
+## Relationship
+
+Below workflow steps are related with this step and should be used as recommended.
+
+### Required Steps
+
+There is no required step that needs to be run beforehand for this step to work as expected.
+
+- [Git Clone](https://docs.appcircle.io/workflows/common-workflow-steps/git-clone)
+- [Android Build for UI Testing](https://docs.appcircle.io/workflows/android-specific-workflow-steps/android-build-for-ui-testing)
```

Comment:
We should also add iOS-related required steps to make it compatible with the steps documents.